src/datasources/floodscan.py

```diff
@@ -52,10 +53,20 @@ def load_floodscan_cogs(
         da_in = process_floodscan_cog(
             cog_name=cog, container_name="global", mode="dev"
         )
-        das.append(da_in)
+        da_sfed = subset_band(da_in, band="SFED")
+        das.append(da_sfed)
+
     return xr.combine_by_coords(das, combine_attrs="drop")
 
 
+def subset_band(da, band="SFED"):
+    long_name = np.array(da.attrs["long_name"])
+    index_band = np.where(long_name == band)[0] + 1
+    da_subset = da.sel(band=index_band)
```

Comment:
One tiny thing- we may want to use `.isel()` here instead, then we don't rely on the bands being actually called `1` and `2`, and kind of better reflects the `np.where()` method's output (so you don't need to add the ` + 1`).

Reply:
nice, i like it - don't know that i fully understand the difference in the `.sel()` and `.isel()` methods, but it does work an looks nicer! [79d5259](https://github.com/OCHA-DAP/hdx-floodscan/pull/1/commits/79d52590d6856b20c8c18538d4da36867d77c46b)